.github/workflows/ci.yml

```diff
@@ -59,14 +61,27 @@ jobs:
               run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
 
             - name: Execute tests (Unit and Feature)
-              run: vendor/bin/phpunit
+              run: vendor/bin/phpunit --log-junit=reports/report-phpunit.xml --coverage-clover=reports/coverage-phpunit.xml
 
             - name: Static analysis with PHPStan
-              run: vendor/bin/phpstan analyse
+              run: vendor/bin/phpstan analyse --error-format=github --error-format=json > reports/phpstan.json
```

Comment:
This won't work. We will just run it twice, as PHPStan caches it results, this will be fast :)